scripts/badger/process_bribes_graviaura.py

```diff
@@ -24,9 +24,9 @@
 PROCESSOR = SAFE.contract(r.aura_bribes_processor, from_explorer=True)
 
 # tokens involved during processing
-WETH = interface.IWETH9(r.treasury_tokens.WETH, owner=SAFE.account)
```

Comment:
Why doing this? These tokens are assigned for their usage after the bribes processing but not expected to come in as bribes themselves. It can occur but its is not the intention for the most. Doesn't matter much tbh just curious on the reasons for the change.

Reply:
because the last couple of weeks i have needed to add new tokens there to ignore, since the autovoter is picking up dust. it is easier to do so with all of them coming from that claimable list